### PR TITLE
[IMP] [mrp_bom_version] updated documentation

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
I have updated the module documentation. In both english and spanish.

Since it is the first time that we're providing documentation in two languages, your feedback is appreciated.

### CODE REVIEW

File: mrp_bom_version/README.rst [hunk 1]
```diff
@@ -82,4 +85,4 @@ Credits
 Contributors
 ------------
 
-** to do **
+* Eficent, www.eficent.com
```
Comment (oihane) [inline mrp_bom_version/README.rst]: If you take a look to [`__openerp__.py` file](https://github.com/odoomrp/odoomrp-wip/blob/8.0/mrp_bom_version/__openerp__.py#L29) you will see that there are some contributors, as listed below:

```
* Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
* Ana Juaristi <anajuaristi@avanzosc.es>
* Alfredo de la Fuente <alfredodelafuente@avanzosc.es>
```

So you can add them in the README.rst file.

**Comment by oihane** [comment]
About the two language README I do not have any idea if they are accepted in OCA. @pedrobaeza please comment.

**Comment by pedrobaeza** [comment]
RST doesn't support multi-language, so we should use another technique to provide a multi-lingual module README.

**Comment by pedrobaeza** [comment]
We can have README.rst in English and README_es.rst for spanish, but until the, I close.

**Comment by JordiBForgeFlow** [comment]
Should I then prepare the two separate files and send a new PR?

**Comment by pedrobaeza** [comment]
Yes, please

**Pull request closed without merging**